minishift/README.md

```diff
@@ -64,15 +64,22 @@ This build uses developer account for creating a project called `planner-service
 
 Above command then automates the process of running the containers on OpenShift in MiniShift by using Kedge.
 
+Now, when you want to use specific tag for WIT image or for AUTH image, use following `optional` values
+
+```
+WIT_IMAGE_TAG=<<WIT_TAG>> AUTH_IMAGE_TAG=<<AUTH_TAG>> make dev-openshift
+```
+When not specified any tag it is `latest` always.
+
 
 ## Cleanup
 When you want to stop all the services running in MiniShift, use following command
 ```
-make dev-planner-openshift-clean
+make dev-openshift-clean
```

Comment:
Or how about
`make clean-openshift`
 https://github.com/fabric8-services/fabric8-auth/pull/207/files#diff-b67911656ef5d18c4ae36cb6741b7965R334

Reply:
indeed! done in https://github.com/fabric8-services/fabric8-wit/pull/1792/commits/ea478b8372fa562f41301e2cbf17a981bf6200e3